tools/GenerateIntegrationDefinitions/Program.cs

```diff
@@ -105,6 +105,16 @@ private static string GetMethodSignature(MethodInfo method)
             var returnType = method.ReturnType;
             var parameters = method.GetParameters().Select(p => p.ParameterType).ToArray();
 
+            var requiredParameterTypes = new[] { typeof(int), typeof(int), typeof(long) };
```

Comment:
Since I was adding another parameter to all wrappers, I though it useful to add this validation.